playbooks/ansible/roles/sit.cephfs/tasks/repo/centos.yml

```diff
@@ -3,7 +3,6 @@
   when: inventory_hostname in groups['cluster']
   block:
     - name: Install VFS module
-      when: config.be.variant == 'vfs'
```

Comment:
Can you please add a comment for this task stating the reason for installing `samba-vfs-cephfs` irrespective of the CephFS backend variant?

Reply:
sure!

Reply:
I was just wondering can there be any possibility of including ceph_snapshots.so as a separate package as othe users with kernel mount essentially would not want to install the vfs module? Or is there any specific reason for this behavior? Please explain if I am missing something. @anoopcs9

Reply:
Packaging _ceph_snapshots.so_ as part of `samba-vfs-cephfs` might have stemmed from the fact that it is currently only built conditionally checking the presence of _libcephfs_ headers([source3/wscript](https://git.samba.org/?p=samba.git;a=blob;f=source3/wscript;h=4048ec5aad50c4be3958e53b7c6f4418e8af4301;hb=HEAD#l1914)). But may be we could make adjustments to have it alongside other common vfs modules. I'll modify spec file to reflect such a change. Either we wait for it or install `samba-vfs-cephfs` always for CephFS.

Reply:
For now I have updated the comment. If it takes time for the change to come into picture, we can add a new PR to update the changed package

Reply:
On a second thought I think its worth waiting for https://github.com/samba-in-kubernetes/samba-build/pull/45 to land so that we avoid the situation with installing `samba-vfs-cephfs` always for CephFS.

Reply:
That also works for me, I will revoke this comment in next change set when the change in samba-build lands